### PR TITLE
feat(plugin): Add developer activity shelf to experiments plugin

### PR DESCRIPTION
**Pull request by happyendermangit**
This plugin enables the hidden developer activity shelf menu, below are some screenshots
image
image

### CODE REVIEW

**Comment by Vendicated** [comment]
probably makes more sense as part of the experiments plugin

**Comment by happyendermangit** [comment]
> experiments

Maybe, i'll make a pull request right now

**Comment by happyendermangit** [comment]
When review?

**Comment by nin0-dev** [comment]
be patient. prs are reviewed when maintainers have the time to and want to, begging is just annoying and won't help

File: src/plugins/experiments/index.tsx [hunk 1]
```diff
@@ -119,5 +120,11 @@ export default definePlugin({
                 Only use experiments if you know what you're doing. Vencord is not responsible for any damage caused by enabling experiments.
             </Forms.FormText>
         </ErrorCard>
-    ), { noop: true })
+    ), { noop: true }),
+    start() {
+        findStore("DeveloperActivityShelfStore").getIsEnabled = () => { return true; };
```
Comment (Vendicated) [inline src/plugins/experiments/index.tsx:125]: do not use monkey patches

**Pull request closed without merging**